allennlp/nn/util.py

```diff
@@ -1289,11 +1289,13 @@ def batched_index_fill(
     filled_target : `torch.Tensor`
         A tensor with shape (batch_size, sequence_length) where 'indices' are filled with `fill_value`
     """
+    target = target.clone()
```

Comment:
I guess that works, but I thought you could just replace `scatter_()` with `scatter()`?

Reply:
Sure